packages/snage/src/config/migrate.ts

```diff
@@ -24,10 +24,12 @@ export const migrate =
         if (!(yamlDoc.contents instanceof YAMLMap)) {
```

Comment:
typo in commit message 
- "migartion" -> "migRAtion"
- "A.rang(2,1)" -> "A.rangE(2,1)"